crates/terminal_view/src/terminal_element.rs

```diff
@@ -755,7 +755,7 @@ impl Element for TerminalElement {
                 // then have that representation be converted to the appropriate highlight data structure
 
                 let (cells, rects) = TerminalElement::layout_grid(
-                    cells,
+                    cells.iter().cloned(),
```

Comment:
@mikayla-maki: this might be the only change I'm wondering if we should go back to what we had before.